src/configs/binary-elementwise-config.c

```diff
@@ -1143,11 +1151,19 @@ static void init_qu8_vadd_config(void) {
     qu8_vadd_config.init.qu8_add = xnn_init_qu8_add_minmax_scalar_params;
     qu8_vadd_config.minmax.element_tile = 4;
   #elif XNN_ARCH_RISCV
```

Comment:
instead of RVV and scalar kernels for riscv, I suggest
#elif XNN_ARCH_RISCV && XNN_ENABLE_RISCV_VECTOR
and there will be an #else below (new) that has scalar.